homework-1/student.py

```diff
@@ -0,0 +1,58 @@
+"""
+
+TASK 2
+
+Write a base class to represent a student. Below is a starter code.
+Feel free to add any more new features to your class.
+As a minimum a student has a name and age and a unique ID.
+
+Create a new subclass from student to represent a concrete student doing a specialization, for example:
+Software Student and Data Science student.
+
+"""
+
+
+class Student:
+
+    def __init__(self, name, age, id):
+        self.name = name
+        self.age = age
+        self.id = id
+        self.subjects = dict()
+
+    def add_subject(self, subject, grade):
+        self.subjects.update({subject: grade})
+
+    def remove_subject(self, subject):
+        self.subjects.pop(subject)
```

Comment:
Might have been nice to consider the case where someone tries to remove a subject that wasn't present